app/models/patient.rb

```diff
@@ -540,8 +540,9 @@ def restricted?
     restricted_at != nil
   end
 
-  def send_notifications?
-    !deceased? && !restricted? && !invalidated?
+  def send_notifications?(team:, send_to_archived: false)
+    !deceased? && !restricted? && !invalidated? &&
+      (send_to_archived || !archived?(team:))
```

Comment:
```suggestion
      (send_to_archived || not_archived?(team:))
```

I think this reads slightly better, but not a big deal.